src/OracleService/OracleService/config.json

```diff
@@ -8,6 +8,9 @@
     "Https": {
       "Timeout": 5000
     },
+    "NeoFs": {
+      "Nodes": []
```

Comment:
We are a NeoFS node, aren't we? Why do we query from other nodes?

Reply:
OK，I will change it.

Reply:
Load Balance
```
Random random = new Random();
            int index = random.Next() % Settings.Default.Fs.FSNodes.Length;
            try
            {
                byte[] res = Get(cancellation, privateKey, uri, Settings.Default.Fs.FSNodes[index]);
                Utility.Log(nameof(OracleFsProtocol), LogLevel.Debug, $"NeoFS result: {res.ToHexString()}");
                return (OracleResponseCode.Success, Convert.ToBase64String(res));
            }
            catch (Exception e)
            {
                Utility.Log(nameof(OracleFsProtocol), LogLevel.Debug, $"NeoFS result: error,{e}");
                return (OracleResponseCode.Error, null);
            }
```

Reply:
We should either query from the neofs node in the same process or connect to localhost.

Reply:
Our c# neofs plugin isn't ready yet. I think we can use localhost.
We can set `localhost` in settings.

Reply:
> Our c# neofs plugin isn't ready yet. I think we can use localhost.

Then we should use localhost. We can remove the settings.

Reply:
OK.Got it.

Reply:
This kind of design is not elegant, a bit hard coding. I think it's better to set the IP and port of a single node in the configuration file instead

Reply:
Then let's use a single node first. We don't need load balance in this case, I think. The next step is to use a single process.

Reply:
OK.Got it.